packages/account/src/App.tsx

```diff
@@ -19,8 +20,10 @@ const App = ({ passthrough }: TAppProps) => {
 
     return (
         <MobxContentProvider store={root_store}>
```

Comment:
@yaswanth-deriv If `connect` is not in use anymore we should be able to remove the `MobxContentProvider` and the whole `Stores` folder in the `account` package including the `initStore` method which seems to be redundant already.